src/Components/Weather/Weather.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC, useState } from "react";
+import { IGetInfo } from "../constants/constants";
+import "antd/dist/antd.css";
+import "./Weather.css";
+import Cards from "../Cards/Cards";
+import { Spin } from "antd";
+import { useEffect } from "react";
+
+const Weather: FC<IGetInfo> = ({ data }) => {
+ 
+  const [isLoaded, setIsLoaded] = useState<boolean>(false);
+  const arrayCityInfoFromStorage = Object.entries(sessionStorage);
+  const filterArrayCityInfoFromStorage = arrayCityInfoFromStorage.map(
+    (el: Array<any>) => [el[0], JSON.parse(el[1])]
+  );
```

Comment:
you can refactor this code shorter, in 1 line, also better to use shorter names of variables